index.html

```diff
@@ -264,7 +264,11 @@ <h3>MathML Element Mappings</h3>
 		  </td>
 		  <td class="axapi">
 		    <span class="role">AXRole: <code>NSAccessibilityGroupRole</code></span><br />
-		    <span class="subrole">AXSubrole: <code>AXMathFraction</code></span>
+		    <span class="subrole">AXSubrole: <code>AXMathFraction</code></span><br />
+		    <span class="attrs">AXAttributes:
+		      <code>AXMathFractionNumerator</code> pointing to a first in-flow child<br />
+		      <code>AXMathFractionDenominator</code> pointing to a second in-flow child
```

Comment:
Nit: "pointing to *a* {first,second} in-flow child" sounds weird to me. Should "a" either be "the" or removed?